apps/designer/app/designer/features/style-panel/shared/css-value-input/css-value-input.tsx

```diff
@@ -412,10 +412,10 @@ export const CssValueInput = ({
     onKeyDown: inputProps.onKeyDown,
   });
 
-  const isCurrentBreakpoint = useIsFromCurrentBreakpoint(property);
+  const isLocalStyle = props.value?.local !== undefined;
   const prefix = icon && (
     <CssValueInputIconButton
-      state={isCurrentBreakpoint ? "set" : undefined}
+      state={isLocalStyle ? "set" : undefined}
```

Comment:
would it be better to pass that state as a prop and so we woun't depend here on StyleValueInfo ?

Reply:
So enum with local and remote?

Reply:
state already enum, currently "set", but yeah we will rename them all

Reply:
"default" or "preset" as last case?